.github/workflows/ci.yml

```diff
@@ -227,9 +227,32 @@ jobs:
       - name: Rename Binary with Target Name
         run: mv target/${{ matrix.target }}/release/tailcall target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
 
+      # Copy the CLI binary and rename it to include the name of the target platform
+      - name: Copy CLI binary
+        id: copy-cli
+        run: |
+          mkdir dist
+          cp target/${{ matrix.target }}/release/tailcall ./dist/tailcall-${{ matrix.target }}
+
       - name: Upload ${{ matrix.target }} Binary
         uses: xresloader/upload-to-github-release@v1
         with:
           release_id: ${{ needs.draft_release.outputs.create_release_id }}
           file: target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
           overwrite: true
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Generate npm packages
+        run: node npm/tailcall/scripts/generate-packages.js
+        env: APP_VERSION=${{ needs.draft_release.outputs.create_release_name }}
+
+      - name: Publish npm packages as latest
+        # CD to package and publish and move back to top-level folder
+        run: for package in npm/*; do cd $package; npm publish --tag latest --access public --provenance; cd ../..; done
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
- Can we not publish everything together? What happens if one of the publish command fails?
- Can we log after every publish?

Reply:
If one of them fails other one continues. 

Logging in the sense? Just printing that one of the package is published right? 

Also by default npm would console everything related to push and  the reason for push fails etc.

Reply:
Good point. npm logs should be enough.

Reply:
but ignoring error in that case could mask the problem as the ci will pass successfully and to know about problem we need to thoroughly read through logs or inspect npm packages

Reply:
we can also add an dry-run npm release workflow in order to able to test that everything works as expected especially before merging this

Reply:
We should use `npm publish --workspaces` command instead of looping using bash.

https://docs.npmjs.com/cli/v7/commands/npm-publish#workspaces